impeller/entity/shaders/tile_mode_utils.glsl

```diff
@@ -0,0 +1,21 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const float kTileModeClamp = 0;
```

Comment:
Please add a comment nearby indicating that these values must correspond to the order of the items in the `Entity::TileMode` enum class.

Reply:
Done.